tox.ini

```diff
@@ -33,6 +33,16 @@ commands =
     black .
     isort .
 
+[testenv:test-all]
```

Comment:
not-blocking: I still do not like the `test-all`, since this will give output for tics, and it will not be used by anything else. I will change it to `tics` or `tics-tests`.

Reply:
Done